tests/asconfig/index.js

```diff
@@ -0,0 +1,56 @@
+const asc = require("../../cli/asc");
+const loader = require("../../lib/loader");
+const args = process.argv.slice(2);
+const fs = require("fs");
+const path = require("path");
+
+/** @type {Uint8Array} */
+let binary;
+/** @type {Uint8Array} */
+let wat;
+asc.main(["assembly/index.ts", "--outFile", "output.wasm", "--textFile", "output.wat", "--explicitStart", ...args], {
```

Comment:
@dcodeIO or @MaxGraey should we have wat output? I added it in to see if it was necessary, but I'm thinking it's extraneous.

Reply:
If we don't ask for a watfile, it seems to output the file to stdout. I am at least leaving it as cli args here.

Reply:
It shouldn't output to stdout if there is at least one other output file, like the binary here. If it does, that'd be a new bug.

Reply:
I decided to remove the watfile output and leave it in because this pull request is a feature request. I will file a separate pull request for this later to try and fix it.

Reply:
Btw, a good alternative to wasm and wat output could be https://github.com/jtenner/assemblyscript/pull/2, comparing fixtures of the expected final configs instead of code.